src/EditorFeatures/Core/Shared/Tagging/TagSources/ProducerPopulatedTagSource.cs

```diff
@@ -63,16 +64,7 @@ internal abstract partial class ProducerPopulatedTagSource<TTag> : TagSource<TTa
 
         private ImmutableDictionary<ITextBuffer, TagSpanIntervalTree<TTag>> _cachedTags;
 
-        private bool _computeTagsSynchronouslyIfNoAsynchronousComputationHasCompleted;
-
-        /// <summary>
-        /// A function that is provided to the producer of this tag source. May be null. In some
-        /// scenarios, such as restoring previous REPL history entries, we want to try to use the
-        /// cached tags we've already computed for the buffer, but those live in a different tag
-        /// source which we need some help to find.
-        /// </summary>
-        private readonly Func<ITextBuffer, ProducerPopulatedTagSource<TTag>> _bufferToRelatedTagSource;
```

Comment:
Removed in https://github.com/dotnet/roslyn/pull/3934 .  Never used in the code.